scripts/spack/packages/axom/package.py

```diff
@@ -31,57 +30,61 @@ def get_spec_path(spec, package_name, path_replacements={}, use_bin=False):
 
 class Axom(CachedCMakePackage, CudaPackage, ROCmPackage):
     """Axom provides a robust, flexible software infrastructure for the development
-       of multi-physics applications and computational tools."""
+    of multi-physics applications and computational tools."""
 
-    maintainers = ['white238']
+    maintainers = ["white238"]
 
     homepage = "https://github.com/LLNL/axom"
-    git      = "https://github.com/LLNL/axom.git"
-    tags     = ['radiuss']
+    git = "https://github.com/LLNL/axom.git"
+    tags = ["radiuss"]
 
-    version('main', branch='main', submodules=True)
-    version('develop', branch='develop', submodules=True)
-    version('0.6.1', tag='v0.6.1', submodules=True)
-    version('0.6.0', tag='v0.6.0', submodules=True)
-    version('0.5.0', tag='v0.5.0', submodules=True)
-    version('0.4.0', tag='v0.4.0', submodules=True)
-    version('0.3.3', tag='v0.3.3', submodules=True)
-    version('0.3.2', tag='v0.3.2', submodules=True)
-    version('0.3.1', tag='v0.3.1', submodules=True)
-    version('0.3.0', tag='v0.3.0', submodules=True)
-    version('0.2.9', tag='v0.2.9', submodules=True)
+    version("main", branch="main")
+    version("develop", branch="develop")
+    version("0.7.0", tag="v0.7.0")
+    version("0.6.1", tag="v0.6.1")
+    version("0.6.0", tag="v0.6.0")
+    version("0.5.0", tag="v0.5.0")
+    version("0.4.0", tag="v0.4.0")
+    version("0.3.3", tag="v0.3.3")
+    version("0.3.2", tag="v0.3.2")
+    version("0.3.1", tag="v0.3.1")
+    version("0.3.0", tag="v0.3.0")
+    version("0.2.9", tag="v0.2.9")
 
-    patch('scr_examples_gtest.patch', when='@0.6.0:0.6.1')
+    # https://github.com/spack/spack/issues/31829
+    patch("examples-oneapi.patch", when="@0.6.1 +examples %oneapi")
```

Comment:
Do we need to add this patch file to axom repo?
I don't see it in the PR files.

Reply:
It exists in the Spack repo